nbrabant/rpi-inventory
# Issue#27 translation module

added i18n module to application
adjusted all parts that it can work with the translations
added first translation on configuration page
added select input to change the language

How to test:
Now one can change the language on the configuration page. After changing the language, you should see the lable for the
language select should change. 
As the default language is set to "en" (you can change this via env-var) at the beginning you see the label "App Language".
After switching you should see "Langue de l'application"

For now this should be ok. Needs some extra styling to be more sexy
